frontend/src/metabase-lib/aggregation.ts

```diff
@@ -35,6 +36,22 @@ export function aggregate(
   return ML.aggregate(query, stageIndex, clause);
 }
 
+export function aggregateByCount(query: Query): Query {
```

Comment:
I decided to introduce this util in `metabase-lib` because it's already needed in 3 places